res/layout/warning_banner_thread_list_item.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ * Copyright 2010 Andrew Shu
```

Comment:
Since you're adding a new file, this is actually copyright you!

If you don't want to put your real name here, you can just change it to 'Diode Contributors'.  Don't forget to change the year to 2017 too.

Reply:
I just copied the file I added in 2015. So I'm updating the comment in the file from 2015 as well.